notebooks/notebook_template.ipynb

```diff
@@ -31,6 +31,8 @@
       "source": [
```

Comment:
<p>I prefer to keep the BUCKET_NAME + BUCKET_URI. I recall in the past some instances were a method takes the bucket name w/o the gs:// prefix -- but I admit I can't at the top of my head think of a specific method that does.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1204/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>They should be able to figure it out if they use such an API.</p>